tests/integration_tests.rs

```diff
@@ -0,0 +1,22 @@
+extern crate meshlite;
+
+use meshlite::mesh::Mesh;
+use meshlite::subdivide::Subdivide;
+
+/// Test added for https://github.com/huxingyi/meshlite/pull/2
+#[test]
+fn verify_cube_subdivision() {
+    let mut mesh = cube();
+    let sub = mesh.subdivide();
+    assert_eq!(26, sub.vertex_count);
+    assert_eq!(24, sub.face_count);
+}
+
+/// Should this be part of the public API? (currently defined in two other places)
```

Comment:
I also think making it public is better, it's a common API. All current public APIs are for Dust3D, if we want meshlite to be a general library, there are much more APIs should be added.

Reply:
Sounds good. Would you like to think about how to structure that some more or should I make a PR?

Reply:
How about add to a separate file such as premitives.rs?

Reply:
Sounds like a good idea.